R/FilterState-utils.R

```diff
@@ -75,6 +78,9 @@ init_filter_state <- function(x,
                               fixed = FALSE,
                               disabled = FALSE,
                               extract_type = character(0),
+                              metadata_id = character(0),
+                              metadata_title = character(0),
```

Comment:
I need to make mental note that `metadata_id` is the variable name and `metadata_title` is the variable label.
I sometime thinks that `metadata_title` is the variable name.

Should we name this `metadata_varname` and `metadata_varlabel` instead to make it consistent with `varname` and `varlabel` variables that's already being used in other places?

Or is there any better naming convention?

Reply:
On second thought, the metadata items are not actually the varname and varlabel. Since we are only using varname and varlabel as fall-back values, I think it's better to have their names better reflect what they are.

Reply:
resolving because we'll handle the metadata stuff on other issue.